README.md

```diff
@@ -89,8 +77,53 @@ The kernel headers are necessary because VFIO structs and defines are reused.
 
 Finally build your program and link with `libvfio-user.so`.
 
-Example
-=======
+Supported features
```

Comment:
Should we also add that currently non-PCI devices are not fully supported?

Reply:
good idea, thanks